db/migrate/20201217163340_update_enterprise_social_links.rb

```diff
@@ -1,11 +1,11 @@
-class UpdateEnterpriseSociaLinks < ActiveRecord::Migration
-  def change
+class UpdateEnterpriseSocialLinks < ActiveRecord::Migration
+  def up
     say_with_time "Updating enterprises..." do
       count = 0
       Enterprise.find_each do |enterprise|
         unless enterprise.instagram.nil?
           say "Update enterprise: #{enterprise.id}"
-          enterprise.update(instagram: enterprise.instagram.downcase.try(:gsub, enterprise.instagram_regex, '\1').delete('@'))
+          enterprise.update(instagram: enterprise.instagram)
```

Comment:
Nice, this removes some code duplication. Now, as counter-intuitive as it may sound, we should add some code duplication to make the migration independent of the changing app code (see [wiki](https://github.com/openfoodfoundation/openfoodnetwork/wiki/Database-migrations)). You can copy parts of the Enterprise model here:
```rb
class Enterprise < ActiveRecord::Base
  INSTAGRAM_REGEX = %r{\A(?:https?://)?(?:www\.)?instagram\.com/([a-zA-Z0-9._@-]{1,30})/?\z}

  def instagram=(value)
    self[:instagram] = value.downcase.try(:gsub, INSTAGRAM_REGEX, '\1').delete('@')
  end
end
```